src/components/common/Chat/index.js

```diff
@@ -39,7 +40,12 @@ const Chat = () => {
             <div className="chat__title">{match.user.full_name || t('profile.anonymous')}</div>
             <div>{match.last_message || t('profile.noMessages')}</div>
           </div>
-          <img className="chat__image" src={match.topic_icon} alt="topic" />
+          <div className="chat__img-container">
+            <img className="chat__image" src={match.topic_icon} alt="topic" />
+            {match.unread_messages ? (
```

Comment:
I'd try using `match.unread_messages &&` instead, just to avoid using the `null`

Reply:
Will do, although since it's a number it would show the 0 so I had to use `!!match.unread_messages` to coerce it into boolean